msi/release.sh

```diff
@@ -33,31 +33,31 @@ test -z "$1" && {
     echo "       where M.m.r.b is the Major, minor, revision, and build number, "
     echo "       respectively."
     echo
-	exit 1
-}
-
-die () {
-	echo "$*" >&2
-	exit 1
-}
-
-function parse_version() {
-	echo "$1" | grep -Px '\d+(\.(0|[1-9]+)){2,3}'
-}
-
-ARCH="$(uname -m)"
-case "$ARCH" in
-i686)
-	BITNESS=32
-	WIX_ARCH=x86
-	;;
-x86_64)
-	BITNESS=64
-	WIX_ARCH=x64
-	;;
-*)
-	die "Unhandled architecture: $ARCH"
-	;;
+    exit 1

```

Comment:
I apologize for the whitespace changes--I can fix this if you want.

Reply:
Thanks